src/commands/gen.ts

```diff
@@ -53,15 +53,24 @@ export default class Gen extends Command {
         await serviceArr[agentCount].init(flags.local);
         if (!flags.local && !serverConfig.disableFluentBitMetrics) {
           // Add fluent bit metrics for every agent
-          serviceArr[agentCount].writeApp({
-            id: `metrics_fluentbit_process`,
-            type: 'metric_s6_process',
-            context: {
-              'app': 'fluentbit',
-              'component': `fluent-bit.${agentCount}`,
-              'childProcess': true,
-            },
-          }, serverConfig, flags.context);
+          if (serverConfig.os === 'linux') {
+            serviceArr[agentCount].writeApp({
+              id: `metrics_fluentbit_process`,
+              type: 'metric_s6_process',
+              context: {
+                'app': 'fluentbit',
+                'component': `fluent-bit.${agentCount}`,
```

Comment:
Should metric_process_windows also have these context settings?

Reply:
Yes